packages/@expo/metro-config/src/transform-plugins/__tests__/__snapshots__/import-export-plugin-snapshots.test.ts.snap

```diff
@@ -145,56 +145,56 @@ exports[`with live bindings transforms export import specifier 1`] = `
 "Object.defineProperty(exports, '__esModule', {
   value: true
 });
-var _apple_icons = require('apple-icons');
 Object.defineProperty(exports, "test", {
   enumerable: true,
   get: function () {
     return _apple_icons.test;
   }
-});"
+});
+var _apple_icons = require('apple-icons');"
 `;
 
 exports[`with live bindings transforms export import specifier and default 1`] = `
 "Object.defineProperty(exports, '__esModule', {
   value: true
 });
-var AppleIcons = _$$_IMPORT_DEFAULT('apple-icons');
-var _apple_icons = require('apple-icons');
-exports.AppleIcons = AppleIcons;
 Object.defineProperty(exports, "test", {
   enumerable: true,
   get: function () {
     return _apple_icons.test;
   }
-});"
+});
+var AppleIcons = _$$_IMPORT_DEFAULT('apple-icons');
```

Comment:
Let's add a `test(AppleIcons)` in the tests. The reason is that if we bump `var AppleIcons = ` to the bottom, we need to ensure that `test(AppleIcons)` comes after, i.e. imports are inserted before the original body but after exports